lib/ansible/modules/utilities/logic/set_extra_var.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright 2016 Pieter Voet <pietervoet@nl.ibm.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+author: "Pieter Voet (pietervoet@nl.ibm.com)"
+module: set_extra_var
+short_description: Sets an extra variable from a playbook.
+description:
+   - This module allows setting new variables.  Variables are set on a global basis
+   - These variables will survive between plays during an Ansible run, even for different host groups
+options:
+    key_value:
```

Comment:
that piece of text is part of a DOCUMENTATION = '''     string...    The 'set_extra_var.py' module is a placeholder ( logic is in the action plugin ) and contains string only. I did not modify it since intital creation, and previously the CI checks did not bomb on this.  
Having sais this, I can imagine the PEP has rules, even for strings..  Will look into this, but while  reading all comments I think that this pull request will probably not get accepted. The problem it tries to solve is acknowledged, but the implementation is favoured to get some place else...